recipes/containerpilot/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "containerpilot" %}
+{% set version = "2.7.8" %}
+{% set sha1= "09158be44c3e887244581d4d019748df9fcfa93c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/joyent/{{ name }}/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha1: {{ sha1 }}
+
+build:
+  number: 0
+  script:
+    - cp {{ name }} $PREFIX/bin
+  skip: True  # [ not linux ]
+
+requirements:
+  build:
```

Comment:
This is a precompiled "go" application similar to conda-forge/consul-feedstock. Hence no external dependencies.

Reply:
Please remove the whole requirements section, then :)

Reply:
done.